src/components/OmouComponents/ReadMore.js

```diff
@@ -0,0 +1,31 @@
+import React,{useState} from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography'
+import theme from "../../theme/muiTheme";
+
+const ReadMore = (props) => {
+
+    const [readMore, setReadMore] = useState(false);
+    const handleReadMoreClick = () => setReadMore(!readMore);
+
```

Comment:
create an explanatory variable for props.children. rename it to: `const bodyText = props.children`